### PR TITLE
Add cisco HwSkus to qos-sai, gb list.

### PR DESCRIPTION
**Pull request by rraghav-cisco**
Adding the cisco GB HwSkus to the ansible/group_vars/sonic/variables
These HwSkus have been used by Cisco and Microsoft for a while now.

@abdosi , @CharudattaSChitale , @gollarharsha , and @jsanghra for review.

### CODE REVIEW

File: ansible/group_vars/sonic/variables [hunk 1]
```diff
@@ -28,7 +28,7 @@ marvell_hwskus: [ "et6448m" ]
 innovium_tl7_hwskus: ["Wistron_sw_to3200k_32x100" , "Wistron_sw_to3200k"]
 
 cisco_hwskus: ["Cisco-8102-C64"]
-cisco-8000_gb_hwskus: ["Cisco-8102-C64"]
+cisco-8000_gb_hwskus: ["Cisco-8102-C64", "Cisco-88-LC0-36FH-M-O36", "Cisco-8101-O8C48", "Cisco-8101-O32"]
```
Comment (abdosi) [inline ansible/group_vars/sonic/variables:31]: please add : Cisco-88-LC0-36FH-O36

**Review by abdosi — CHANGES_REQUESTED** [review]
as comment

**Comment by abdosi** [comment]
please rebase.

File: ansible/group_vars/sonic/variables (hunk 1 above)
Comment (rraghav-cisco) [reply ansible/group_vars/sonic/variables]: @abdosi : Done.

**Comment by rraghav-cisco** [comment]
@abdosi : resolved conflicts.

**Comment by rraghav-cisco** [comment]
The azp job failed in a mux-related code, not related to this PR.

**Comment by rraghav-cisco** [comment]
@abdosi : Re-updated PR, all pre-checks have passed. Pls merge this.

**Review by abdosi — APPROVED** [review]

**Comment by rraghav-cisco** [comment]
> @rraghav-cisco PR conflicts with 202205 branch
@mssonicbld : Raised for 202205: https://github.com/sonic-net/sonic-mgmt/pull/10822

**Pull request merged**